dom.bs

```diff
@@ -780,6 +780,7 @@ flags that are all initially unset:
  <li><dfn export for=Event id=initialized-flag>initialized flag</dfn>
  <li><dfn export for=Event id=dispatch-flag>dispatch flag</dfn>
  <li><dfn export for=Event id=in-passive-listener-flag>in passive listener flag</dfn>
+ <li><dfn export for=Event id=found-listener-flag>found listener flag</dfn>
```

Comment:
I was thinking, if "inner invoke" / "find" just returns a boolean, we don't need extra state on the event. I think that would be better.